pipelines/ribomsqc/testdata/samplesheet.csv

```diff
@@ -0,0 +1,2 @@
+id,raw_file
+BSA_QC_test,BSA_QC_test.raw
```

Comment:
Think you want the raw github path here right?

Reply:
yep, should be like this? 

```
id,raw_file
BSA_QC_test,https://raw.githubusercontent.com/nf-core/test-datasets/ribomsqc/testdata/BSA_QC_test.raw
```